conferences/abstracts/RSECon19-BinderHub-workshop-application.md

```diff
@@ -0,0 +1,96 @@
+
+### Abstract
+
+_(1250 characters max, current count: 1244 [please update if you edit])_
+
+BinderHub is a Cloud-based, multi-user server technology that captures the computing environment of a GitHub repository within a Docker image.
+It then provides an interactive browser from which code can be run in the environment under which it was developed without placing installation responsibilities on the user.
+A unique URL to this browser is generated making the software easily sharable with collaborators.
+BinderHubs can be built anywhere in the world and are Cloud-neutral.
+Building a locally hosted BinderHub is advantageous as it would facilitate the use of private repositories and make available a greater variety of computing resources - facilities which are not currently available on the public Binder service.
+
+By harnessing the power of Cloud computing and building a BinderHub with expanded capabilities, scientific analyses can become more accessible and transparent to the wider community.
+BinderHubs can showcase the reproducibility of software as well as being a valuable teaching or demonstration tool.
+
+In this workshop, attendees will learn how to deploy a BinderHub on Microsoft Azure and connect with others who have or are interested in doing so.
+Attendees should be familiar with Azure Cloud computing and the command line.
+
+### Audience
+
+_Would your target audience be required to have any prerequisite skills/background knowledge e.g. knowledge of a particular language?_
+
+Attendees would preferably be familiar with Microsoft Azure and the command line.
+
+### Outcomes
+
+_How will your attendees benefit from your session? What do you expect them to gain/learn?_
+
+_(600 characters max, current count: 524 [please update if you edit])_
+
+Attendees to this workshop will learn how to deploy a BinderHub on Microsoft Azure Cloud platform.
+They will learn the multiple applications of a BinderHub as a teaching or demonstration tool; the technology powering a BinderHub; and why reproducibility is important in research.
+Attendees will be provided with enough knowledge to make an informed decision on whether building their own BinderHub is advantageous to their goals and connect with other RSEs/researchers who have build a BinderHub or are interested in doing so.
+
+### Accessibility
+
+_How will you ensure that your content will be accessible to the diverse audience of the conference?_
+
+### Promotion
+
+_How will you attract a broad and diverse audience for your session?_
+
+I hope to attract attendees by being a visible, active and welcoming member of the RSE community.
+I especially hope that by giving this workshop, I encourage more women, members of the LBGTQ+ and non-binary communities to engage with Cloud computing projects and open source communities.
+
+### Mentorship
+
+_Optional section._
+
+_Would you like a mentor? If so then what would you like the mentor to do?_
+
+No?
```

Comment:
My 2cents on this is that it would be nice to have a mentor there, unless you're confident that you have helpers who will be there to answer questions etc. Seems like a good opportunity to get some support from an experienced member of the RSE community.

Doesn't matter too much though!